packages/react-native-icons/convert.js

```diff
@@ -69,9 +69,9 @@ function processFiles(src, dest) {
   // Finally add the interface definition and then write out the index.
   indexContents.push('export { FluentReactNativeIconsProps } from \'./utils/FluentReactNativeIconsProps.types\'');
   indexContents.push('export { default as wrapIcon } from \'./utils/wrapIcon\'');
-  indexContents.push('export { default as bundleIcon } from \'./utils/bundleIcon\'');
+  //indexContents.push('export { default as bundleIcon } from \'./utils/bundleIcon\'');
```

Comment:
Should these commented lines be removed?

Reply:
Not for now. Maybe I'll need it later. If not, I'll remove it